File: VB/WorkflowServerService2/Program.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.ServiceProcess
Imports System.Text

Namespace WorkflowServerService2
	Friend Module Program
		''' <summary>
		''' The main entry point for the application.
		''' </summary>
		Sub Main()
			Console.WriteLine("Starting...")
			Dim service As New WorkflowServerService2()
			service.Start()
			Console.WriteLine("Started. Press Enter to stop.")
			Console.ReadLine()
			Console.WriteLine("Stopping...")
			service.Stop()
			Console.WriteLine("Stopped.")
		End Sub
	End Module
End Namespace

File: CS/WorkflowServerService2/Program.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.ServiceProcess;
using System.Text;

namespace WorkflowServerService2 {
    static class Program {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        static void Main() {
            Console.WriteLine("Starting...");
            WorkflowServerService2 service = new WorkflowServerService2();
            service.Start();
            Console.WriteLine("Started. Press Enter to stop.");
            Console.ReadLine();
            Console.WriteLine("Stopping...");
            service.Stop();
            Console.WriteLine("Stopped.");
        }
    }
}
